packages/grafana-ui/src/components/Panel/Panel.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
   onToggle?: (isOpen: boolean) => void;
 }
 
-export default class Panel extends PureComponent<Props> {
+export class Panel extends PureComponent<Props> {
```

Comment:
@dprokop -- any thoughts on using the name "Panel" here?  Panel is grafana usually means the panel plugins.

This seems similar to `Collapse` in ant.design:
https://ant.design/components/collapse/

Reply:
I totally agree, must have missed this one when moving the styles and Panel is not a good name but `Collapse` seems perfect.

Reply:
I agree too, `Collapse` is 👌